packages/cli/src/cmds/beacon/options.ts

```diff
@@ -8,6 +8,8 @@ interface IBeaconExtraArgs {
   forceGenesis?: boolean;
   genesisStateFile?: string;
   weakSubjectivityStateFile?: string;
+  fetchChainSafeWeakSubjecitivtyState: boolean;
+  weakSubjectivityCheckpoint?: string;
```

Comment:
you should allow passing custom server url and also check if it returns expected response